#goal - 
geometry_msgs/Pose2D relative_goal_pose_2d
---
#result
geometry_msgs/Pose2D final_pose
---
#feedback
float32 start_angle_turned
float32 distance_moved
float32 final_angle_turned